stories/example-screen-staff-list.stories.js

```diff
@@ -289,322 +289,105 @@ const tabsDoubleRowHTML = `
       </a>
     </li>
   </ul>
-  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
-    <li class="nav-item">
-      <a class="nav-link active fs-5" href="#">
+
+  <ul class="nav nav-tabs bg-white ps-7 pe-7">
+    <li class="nav-item py-2 px-3">
+      <a class="fs-6 text-primary" href="#" style="font-weight: 400;">
         <span>All Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" aria-current="page" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>Recommended Jobs</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>My Favorites</span>
       </a>
     </li>
-    <li class="nav-item">
-      <a class="nav-link fs-5" href="#">
+    <li class="nav-item p-2">
+      <a class="fs-6 text-muted" href="#" style="font-weight: 400;">
         <span>Applications</span>
       </a>
     </li>
   </ul>
 `;
 
+const batchAndPagination = `
+  <div class="d-flex align-items-center justify-content-between flex-wrap mx-3 mt-4 mb-3">
+    <div class="d-flex align-items-center justify-content-center">
+      <select class="form-select fs-6" aria-label="Default select example" style="padding-right: 2rem; line-height: 1.3;">
+        <option selected>Batch Options</option>
+        <option value="1">One</option>
+        <option value="2">Two</option>
+        <option value="3">Three</option>
+      </select>
+      <span class="fs-6 text-muted ms-3" style="min-width: 120px;">1 item selected</span>
```

Comment:
We shouldn't be using any inline `min-width` declarations. All 2-D layout should be handled with `flex-box` classes.

In this case maybe it's ok to simply remove this.